server/routes/index.ts

```diff
@@ -14,17 +14,29 @@ interface IndexTemplateValues {
   needAttentionPagination: Pagination
 }
 
-export default function routes({ auditService }: Services): Router {
+export default function routes({ auditService, personRecordService }: Services): Router {
   const router = Router()
 
   router.get('/', async (req, res, next) => {
+    const { token } = res.locals.user
+    const rows: Row[] = []
+
+    const clusters = await personRecordService.getClusters(token)
+
+    clusters.forEach(cluster => {
+      const composition = `${cluster.recordComposition.commonPlatform},
+       ${cluster.recordComposition.delius},
+       ${cluster.recordComposition.libra},
+       ${cluster.recordComposition.nomis}`
+
+      rows.push(Row(LinkItem(cluster.uuid, cluster.uuid), TextItem(composition)))
+    })
```

Comment:
```suggestion
     clusters.forEach({recordComposition,uuid} => {
      const composition = `${recordComposition.commonPlatform},
       ${recordComposition.delius},
       ${recordComposition.libra},
       ${recordComposition.nomis}`

      rows.push(Row(LinkItem(uuid, uuid), TextItem(composition)))
    })
```

Reply:
Ugh hate this interface. I was trying to suggest a bit of destructuring here in the `forEach` rather than writing `cluster.` on each line

Reply:
Strange that this has been added without a test for it - we should try to stick to TDD where we can